uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/groupby/service/go/GOServiceTest.java

```diff
@@ -27,11 +29,21 @@ void setUp() {}
 
     @Test
     void getChildren() {
-        when(goClient.getChildren(ID)).thenReturn(result);
+        when(goClient.getChildren(ID)).thenReturn(resultChildren);
 
         List<GoRelation> children = goService.getChildren(ID);
 
         verify(goClient).getChildren(ID);
-        assertSame(result, children);
+        assertSame(resultChildren, children);
+    }
+
+    @Test
+    void getGoRelation() {
```

Comment:
Do you think a test for not found should be added? 
Do you think the same would also apply for getChildren?

Reply:
I don't think we need separate tests since empty list and empty optional scenarios are covered in the written tests. (mocked results can get one of any form)